### PR TITLE
Changelogs for lakeFSFS and spark metadata client

### PR DESCRIPTION
**Pull request by itaiad200**

### CODE REVIEW

File: clients/hadoopfs/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 ## _Upcoming_
 
+## 0.1.11
+
+What's new:
+* Upgrade lakeFS client to v0.91.0
+* Use CopyObject instead of StageObject
```
Comment (arielshaqed) [inline clients/hadoopfs/CHANGELOG.md:9]:
```suggestion
* Prefer to use CopyObject instead of StageObject if possible.
```

**Review by arielshaqed — APPROVED** [review]
THNAKS!

File: clients/hadoopfs/CHANGELOG.md (hunk 1 above)
Comment (N-o-Z) [inline clients/hadoopfs/CHANGELOG.md:9]:
```suggestion
* Use CopyObject instead of StageObject in rename operation
```

**Review by N-o-Z — APPROVED** [review]
One suggestion

**Pull request merged**